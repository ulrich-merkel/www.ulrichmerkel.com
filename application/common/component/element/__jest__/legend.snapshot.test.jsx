/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ComponentToBeTested from './../legend';

describe('common/component/element/legend', function () {
    it('should render correctly', function () {
        const tree = renderer.create(
            <ComponentToBeTested className='legend'>
                Legend Children
            </ComponentToBeTested>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
