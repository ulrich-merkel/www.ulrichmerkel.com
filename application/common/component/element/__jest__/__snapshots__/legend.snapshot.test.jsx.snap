exports[`common/component/element/legend should render correctly 1`] = `
<legend
  className="m-form__legend legend">
  Legend Children
</legend>
`;
